pkg/systemd/services/services.jsx

```diff
@@ -738,7 +758,8 @@ class ServicesPageBody extends React.Component {
             const unit_id = path[0];
             const get_unit_path = (unit_id) => this.path_by_id[unit_id];
             const unit_path = get_unit_path(unit_id);
-            const unit = this.state.unit_by_path[unit_path];
+            const unit = { ...this.state.unit_by_path[unit_path] };
```

Comment:
This seems unrequired, the previous line should do the same no?

Reply:
Ok, passing it as a prop.